charts/stable/frigate/questions.yaml

```diff
@@ -6,9 +6,21 @@ questions:
 # Include{global}
 # Include{workload}
 # Include{workloadDeployment}
+
 # Include{replicas1}
 # Include{podSpec}
 # Include{containerMain}
+                                - variable: imageSelector
```

Comment:
I can’t recall but probably needs to be in the container level and not at the top

Reply:
I'm basing it off this -> https://github.com/truecharts/charts/blob/4d09adbf6796904df16bab74fa88abd64cfdca65/charts/stable/jupyter/questions.yaml#L12

Reply:
nvm, it was looking like it had no identation on mobile.